specification/semantic_conventions.md

```diff
@@ -0,0 +1,27 @@
+# Semantic Conventions
+
+This document defines the OpenTelemetry and Splunk specific standard attributes for Splunk distributions of OpenTelemetry.
+
+## OpenTelemetry Resource Attributes
+
+**Description:** Required and recommended OpenTelemetry [resource semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk).
+
+All Splunk distributions of OpenTelemetry,
+
+- MUST set the following OpenTelemetry resource attributes according to the [OpenTelemetry Semantic Conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk):
+  - `telemetry.sdk.name`
+  - `telemetry.sdk.version`
+  - `telemetry.sdk.language`
+
+- SHOULD set the following resource attributes when applicable:
+  - `telemery.auto.version`
```

Comment:
```suggestion
- MUST set the following OpenTelemetry resource attributes according to the [OpenTelemetry Semantic Conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.3.0/specification/resource/semantic_conventions#telemetry-sdk).
```

We can get rid of copy-paste of the specs by can just making a "pinned" hyperlink 😉

Reply:
What if the spec changes and add new fields? That'd change GDI spec on the fly without even a version number change :) I don't see any much harm in explicitly mentioning the required fields for Splunk GDI spec and linking to otel for details.

Reply:
Take notice the `tree/v1.3.0`  in the hyperlink 😉

Reply:
Then it won't change and mentioning the attributes inline has no risk so I'd make it easier for the readers and not have them open another page to read 3 words :)